src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -1078,7 +1084,7 @@ private static VersionInfo GetVersionInfo()
                 //verWrittenCur: 0x0001000A, // Added ForceVector in Range
                 //verWrittenCur: 0x0001000B, // Header now retained if used and present
                 //verWrittenCur: 0x0001000C, // Removed Min and Contiguous from KeyType, and added ReadMultilines flag to OptionFlags
-                verWrittenCur: 0x0001000D, // Added escapeChar option and decimal marker option to allow for ',' to be a decimal marker
+                verWrittenCur: 0x0001000D, // Added escapeChar and decimalMarker chars and imputeEmptyFloats flag
```

Comment:
>0x0001000D [](start = 31, length = 10)

Did you intend to increment the version?

Reply:
I don't think it's necessary to increment the version, since the option I'm adding is saved as part of the OptionFlags, which in turn are saved as simply 2 bytes. Since adding the ImputeEmptyFloats doesn't change the saving or loading logic, then I didn't see any reason to update the version.

Is there any reason I'm missing, and should I change the version?

---
In reply to: [436503164](https://github.com/dotnet/machinelearning/pull/5198#discussion_r436503164) [](ancestors = 436503164)

Reply:
Wait, I've just seen your comment down below on the loading logic needing to change a check based on the version of the model. I'll do that, and also add a test saving and loading a TextLoader to check it works :smile:

---
In reply to: [436786694](https://github.com/dotnet/machinelearning/pull/5198#discussion_r436786694) [](ancestors = 436786694,436503164)

Reply:
Ok, I've seen your comment bellow, I think you're right. I'll update this.

---
In reply to: [436786694](https://github.com/dotnet/machinelearning/pull/5198#discussion_r436786694) [](ancestors = 436786694,436503164)

Reply:
Done.

---
In reply to: [436802879](https://github.com/dotnet/machinelearning/pull/5198#discussion_r436802879) [](ancestors = 436802879,436786694,436503164)